openaps/vendors/medtronic.py

```diff
@@ -441,9 +441,6 @@ class set_clock (InputProgramRequired):
   """ Set clock.
 
 
-  -------------------
```

Comment:
Consider:
```
Example
-------------
```

There's two things happening here: one is argparser is using this to format the help output, second, is sphinx will use any RST it finds as additional documentation.  Notice how the `-h` help output corresponds.  What do you think?